README.md

```diff
@@ -245,36 +254,21 @@ Feel free to check the [issues page](../../issues/).
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+If you like this project hit the ⭐️!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+“A recipe has no soul. You as the cook must bring soul to the recipe.”
+– Thomas Keller
```

Comment:
Just so you know, the MIT license link provided below is invalid. Kindly fix it 😉